examples/helloworld/README.rst

```diff
@@ -0,0 +1,75 @@
+==================
+Helloworld Example
+==================
+
+**Requires: python3**
+
+This example creates a two activities and one decider for those activities. The first activity will
+ask for a name and then the second activity will print that name.
+
+How To
+------
+
+Preconfiguration
```

Comment:
This is great stuff! I agree that botoflow is missing examples, but i feel like the "preconfiguration" bits are universal and maybe should move to to the docs/ into a Getting Started or something like that section.
What do you think? And by the way this comment isn't to say that this needs to happen now.